src/main/java/com/sawyereffect/steps/StartingSteps.java

```diff
@@ -14,22 +9,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.IOException;
 import java.util.concurrent.TimeUnit;
 
-public class StartingSteps {
+public class StartingSteps extends DriverFactory{
```

Comment:
The way the framework was designed was so that Steps would be the ones extending DriverFactory to get all the needed objects preloaded (driver in this case) from a single point otherwise what would be needed is to change the architecture of the framework to use a delegation approach.

That change is not part of this change and would require a different issue to be created so it could be accomplished there not on this pull request.